src/agent/ncp.hpp

```diff
@@ -131,11 +131,12 @@ class Controller : public EventEmitter
     /**
      * This method creates a NCP Controller.
      *
-     * @param[in]   aInterfaceName  A string of the NCP interface name.
-     * @param[in]   aRadioUrl       The URL describes the radio chip.
+     * @param[in]   aInterfaceName          A string of the NCP interface name.
+     * @param[in]   aBackboneInterfaceName  The Backbone network interface name.
+     * @param[in]   aRadioUrl               The URL describes the radio chip.
      *
      */
-    static Controller *Create(const char *aInterfaceName, const char *aRadioUrl);
+    static Controller *Create(const char *aInterfaceName, const char *aBackboneInterfaceName, const char *aRadioUrl);
```

Comment:
put `aBackboneInterfaceName` in the end with default nullptr for non BBR usage?

Reply:
Thought about this but decided to put the two arguments for interface names close to each other.
I think it's not that meaningful to use default arguments since it's just called by `otbr-agent` at one place.

Reply:
revised accordingly